frontend/gateway/pb/caps.go

```diff
@@ -0,0 +1,48 @@
+package moby_buildkit_v1_frontend
+
+import "github.com/moby/buildkit/util/apicaps"
+
+var Caps apicaps.CapList
+
+// Every backwards or forwards non-compatible change needs to add a new capability row.
+// By default new capabilities should be experimental. After merge a capability is
```

Comment:
Small precision: `Once a new capability is merged in the source code's master branch, it is considered immutable.`